extensions/spark/kyuubi-spark-connector-hive/src/main/scala/org/apache/kyuubi/spark/connector/hive/read/HiveScan.scala

```diff
@@ -54,6 +54,14 @@ case class HiveScan(
 
   private val partFileToHivePartMap: mutable.Map[PartitionedFile, HivePartition] = mutable.Map()
 
+  override def isSplitable(path: Path): Boolean = {
+    catalogTable.provider.map(_.toUpperCase(Locale.ROOT)).exists {
```

Comment:
Does this support non-datasource tables, such as hive serde ?

https://github.com/apache/spark/blob/531ec8bddc8dd22ca39486dbdd31e62e989ddc15/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala#L212-L216

Reply:
Updated